packages/wdio-utils/src/test-framework/testFnWrapper.ts

```diff
@@ -60,10 +63,14 @@ export const testFrameworkFnWrapper = async function (
     { beforeFn, beforeFnArgs }: BeforeHookParam<unknown>,
     { afterFn, afterFnArgs }: AfterHookParam<unknown>,
     cid: string,
-    repeatTest = 0
+    repeatTest = 0,
+    hookName?: string
 ) {
     const retries = { attempts: 0, limit: repeatTest }
     const beforeArgs = beforeFnArgs(this)
+    if (type !== 'Test') {
```

Comment:
Let's explicitly add it to the functions we want instead of adding it to everything but Test

Reply:
i'm not sure if i understood what you meant, or know how implement.

do you mean that i'll need to remove the if, so it would work with `type==='Test'` as well? or that when we push to this to `afterArgs/beforeArgs` the hookName will be inserted as well, as part of line 70. 

if it's the second one, i'm a bit stumped here, since i can't quite understand how to do this (help wanted 🙂)

Reply:
changed the comparison to look for `Hook`, since we want to implement this on all hooks